lib/internal/tls/secure-context.js

```diff
@@ -261,7 +261,7 @@ function configSecureContext(context, options = {}, name = 'options') {
     context.setSessionIdContext(sessionIdContext);
   }
 
-  if (pfx !== undefined) {
+  if (pfx != null) {
```

Comment:
This would make it look like less of a mistake (using loose quality instead of strict equality) since strict equality is used more often than not:

```suggestion
  if (pfx !== undefined && pfx !== null) {
```

Reply:
I quite like `!= null` pattern, especially since optional chaining and null coalescing are a thing 🤷‍♂️

Reply:
Agreed. I prefer the same because strict equality more verbose. I was following the previous PR in the issue... So I'll include the strict for `ciphers` option also.